microshift_release_notes/microshift-4-12-release-notes.adoc

```diff
@@ -221,3 +221,12 @@ Issued: 2023-05-23
 {product-title} release 4.12.18 is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHBA-2023:3211[RHBA-2023:3211] advisory. The images that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2023:3208[RHBA-2023:3208] advisory.
 
 For the `TopoLVM image`, see link:https://catalog.redhat.com/software/containers/lvms4/topolvm-rhel8/63972de3adcb55263891b983?container-tabs=dockerfile[lvms4/topolvm-rhel8].
+
+[id="microshift-4-12-19-dp"]
+=== RHBA-2023:3290 - {product-title} 4.12.19 bug fix and security update
+
+Issued: 2023-05-31
+
+{product-title} release 4.12.19 is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHBA-2023:3290[RHBA-2023:3290] advisory. The images that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2023:3287[RHSA-2023:3287] advisory.
```

Comment:
```suggestion
{product-title} release 4.12.19 is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHBA-2023:3290[RHBA-2023:3290] advisory. The list of images included in the update are shown in the link:https://access.redhat.com/errata/RHSA-2023:3287[RHSA-2023:3287] advisory.
```

Reply:
@opayne1 One minor nit for your consideration.

Reply:
Roger that, @opayne1 ! Now I wish I'd just said LGTM! :+1:

Reply:
No worries! Always looking to improve clarity! 😄